torch/utils/_pytree.py

```diff
@@ -56,14 +57,38 @@ def _tuple_flatten(d: Tuple[Any, ...]) -> Tuple[List[Any], Context]:
 def _tuple_unflatten(values: List[Any], context: Context) -> Tuple[Any, ...]:
     return tuple(values)
 
+def _namedtuple_flatten(d: NamedTuple) -> Tuple[List[Any], Context]:
+    return list(tuple(d)), type(d)
```

Comment:
You can't just call list on a namedtuple? You do that in the tests no?

Reply:
Good question. Turns out that you can:
```
from collections import namedtuple
Point = namedtuple('Point', ['x', 'y'])
somewhere = Point(x=0, y=1)
print(list(somewhere))
```